okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -592,7 +592,8 @@ private void proxyResponseImmediate(ProxyContext pc, ReadStream<Buffer> res,
         for (HttpClientRequest r : clientRequestList) {
           r.write(data);
         }
-        ctx.response().write(data);
+        res.pause();
```

Comment:
res should be renamed to stream or readStream

Reply:
Shouldn't be a problem.